### PR TITLE
refactor: [M3-6357] - Refactor LinodeCLIModal

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
MUI v5 migration for the `LinodeCLIModal` component.

## Major Changes 🔄
- Ran `tss-react` codemod on `LinodeCLIModal`
- Replaced `default export` with `named export`.

## How to test 🧪
1. Verify e2e tests pass
2. Verify unit tests pass
3. Inspect for any visual regressions

### CODE REVIEW

File: packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx [hunk 1]
```diff
@@ -58,30 +13,66 @@ interface Props {
 
 export type CombinedProps = Props;
```
Comment (abailly-akamai) [inline packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx:14]: `CombinedProps` is unnecessary here.

just export `ImageUploadSuccessDialogProps` above

export interface ImageUploadSuccessDialogProps {
  isOpen: boolean;
  onClose: () => void;
  command: string;
  analyticsKey?: string;
}

**Comment by bnussman** [comment]
Your PR has a changeset ✅

File: packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx [hunk 2]
```diff
@@ -58,30 +13,66 @@ interface Props {
...
+  label: 'StyledLinodeCLIModal',
+})(({ theme }) => ({
+  width: '100%',
+  padding: `${theme.spacing()} ${theme.spacing(2)}`,
+  '& [data-qa-copied]': {
+    zIndex: 2,
+  },
+}));
+
+const StyledCommandDisplay = styled('div')(({ theme }) => ({
```
Comment (abailly-akamai) [inline packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx:49]: always a good idea to add a label for every Styled component. Makes debugging/development easier

**Review by abailly-akamai — COMMENTED** [review]
Looks good! couple comments

File: packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx (hunk 1 above)
Comment (carrillo-erik) [reply packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx]: Good catch, it appears at some point in the past there were additional props which called the need for `CombinedProps`. That's no longer the case and I've made the changes you suggested.

File: packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx (hunk 2 above)
Comment (carrillo-erik) [reply packages/manager/src/components/LinodeCLIModal/LinodeCLIModal.tsx]: I've updated the remaining Styled Components to include labels.

**Review by jaalah-akamai — APPROVED** [review]

**Review by abailly-akamai — APPROVED** [review]

**Pull request merged**